addrmgr/addrmanager_test.go

```diff
@@ -202,7 +208,7 @@ func TestAddAddressByIP(t *testing.T) {
 	amgr = New(dir, nil)
 	amgr.Start()
 	if ka := amgr.GetAddress(); ka == nil {
-		t.Errorf("Address Manager should contain known address")
+		t.Fatalf("Address Manager should contain known address")
```

Comment:
This (and several other cases below) can be `Fatal` rather than `Fatalf` since no formatting is required.